src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,10 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comment = [] }) => (
+  comment.map(com => (
+    <div className="CommentList">
```

Comment:
The key attribute should be here. If you want to wrap the elements inside, consider wrapping the entire map function to avoid creating a div with CommentList class for each element in the array